dired.py

```diff
@@ -1120,7 +1124,16 @@ def run(self, edit, project_folder=False):
                     subprocess.Popen(['sublime'] + items, cwd=self.path)
 
         def run_on_new_window():
-            sublime.active_window().run_command("dired", {"immediate": True, "project": True, "other_group": "left"})
+            settings = sublime.load_settings('dired.sublime-settings')
+            open_on_jump = settings.get('dired_open_on_jump', 'left')
+
+            if open_on_jump:
+                options = {"immediate": True, "project": True}
+
+                if is_string(open_on_jump):
```

Comment:
Value from settings would be Unicode. You should `if open_on_jump == 'left':`

Also, please, add setting into [default settings file](https://github.com/aziz/SublimeFileBrowser/blob/master/dired.sublime-settings) with description.